routes/api/gardenplants.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const express = require('express');
+const HttpStatus = require('http-status-codes');
+
+const interceptors = require('../interceptors');
+const models = require('../../models');
+
+const router = express.Router();
+
+router.get('/', async function(req, res) {
+  const gardenplants = await models.GardenPlant.findAll({
+    order: [['GardenId', 'ASC'], ['PlantId', 'ASC']],
+    include: [models.Garden, models.Plant]
```

Comment:
When fetching GardenPlant, I tell sequelize to INCLUDE the Garden and Plant data that it references.